src/test/java/teammates/test/cases/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -410,14 +410,19 @@ private void testCancelEditQuestion() {
         int qnIndex = 1;
         String qnTextOriginal = "mcq qn";
 
-        // Add question 2 first
+        // Add 2 questions
         feedbackEditPage.clickNewQuestionButton();
         feedbackEditPage.selectNewQuestionType("MCQ");
         feedbackEditPage.fillQuestionTextBoxForNewQuestion(qnTextOriginal);
         feedbackEditPage.fillQuestionDescriptionForNewQuestion("more details");
         feedbackEditPage.fillMcqOptionForNewQuestion(0, "Choice 1");
         feedbackEditPage.fillMcqOptionForNewQuestion(1, "Choice 2");
         feedbackEditPage.clickAddQuestionButton();
+        feedbackEditPage.clickNewQuestionButton();
+        feedbackEditPage.selectNewQuestionType("TEXT");
+        feedbackEditPage.fillQuestionTextBoxForNewQuestion(qnTextOriginal + " 2");
```

Comment:
You might want to change the text of `qnTextOriginal` since it does not make much sense when question 2 is a text question, not a mcq question.